Ali | Amir / Storybook for new mobile components (#1986)
binary-com/deriv-app: packages/components/stories/relative-datepicker/README.md

# Relative DatePicker Component

> Renders a native date picker for mobile. When user clicks on title, the native date picker of mobile will show up. (View this page in mobile device for correct functionality)

Reviewer: ```suggestion
Renders a native date picker for mobile that returns value in days relative to current date. When user clicks on title, the native date picker of mobile will show up. (View this page in mobile device for correct functionality)
```

Author: Corrected.

#### Supported Gestures:

-   Click

> 

Reviewer: Can you add supported events here too?

```suggestion

#### Supported events:
- onChange

```

Author: Added.

## Usage

```jsx
import RelativeDatepicker from 'Components/relative-datepicker';

const DummyComponent = props => (
   <RelativeDatepicker
        onChange={(date) => { console.log(date) }}
        min={0}
        max={5}
        title={'Pick a date'}
    />
);
```

## Props

| Name                          | Type               | Default   | Description                                            |
| ----------------------------- | ------------------ | --------- | ------------------------------------------------------ |

> | onChange                      | {function}         | undefined | Function returns the selected date                     |

Reviewer: ```suggestion
| onChange                      | {function}         | undefined | Function returns value in amount of days, relative to date selected and current date.   |
```

Author: Corrected.

| title                         | {string}           | null      | Title of the date picker                               |

> | min                           | {string \| number} | 0         | Minumim selectable date offset                         |

Reviewer: ```suggestion
| min                           | {string \| number} | 0         | Minimum selectable date offset                         |
```

Author: Corrected.
